debian/rules

```diff
@@ -84,11 +84,10 @@ override_dh_install:
 	-rm debian/tmp/usr/lib/*/libmirprotobuf.so
 ifneq ($(filter xenial artful,$(DEB_DISTRIBUTION)),)
 	dh_install -pmir-utils usr/bin/mirvanity
-	-rm -f debian/tmp/usr/bin/mirvanity
-	dh_install
 else
-	dh_install
+	-rm -f debian/tmp/usr/bin/mirvanity
```

Comment:
This is wrong: mirvanity should only be built on xenial & artful, if it exists elsewhere that is an error that needs correcting, not hiding.